app/backend/create-server.ts

```diff
@@ -1,7 +1,7 @@
 import Debug from 'debug';
 import { createServer as _createServer } from 'http';
 import dotenv from 'dotenv';
-import config from '../config.js';
+import config from '../config/index.js';
```

Comment:
```suggestion
import config from '../config;
```
Does this work?

Reply:
yes, when it's folder then index.js is default
https://nodejs.org/api/modules.html : 
```require('./some-library') would attempt to load:

./some-library/index.js
./some-library/index.node```